### PR TITLE
links to graph operators

### PR DESCRIPTION
**Pull request by mirnawong1**
per [slack therad](https://dbt-labs.slack.com/archives/C05FWBP9X1U/p1702275981683689), this PR spells out the graph operators and links to their links so users don't have to manually search for them. this pr also turns the examples into tabs for easier scrolling/navigation.

### CODE REVIEW

File: website/docs/reference/node-selection/syntax.md [hunk 1]
```diff
@@ -35,14 +35,18 @@ To follow [POSIX standards](https://pubs.opengroup.org/onlinepubs/9699919799/bas
...
 
 Select resources to build (run, test, seed, snapshot) or check freshness: `--select`, `-s`
 
 ### Examples
 
 By default, `dbt run` will execute _all_ of the models in the dependency graph. During development (and deployment), it is useful to specify only a subset of models to run. Use the `--select` flag with `dbt run` to select a subset of models to run. Note that the following arguments (`--select`, `--exclude`, and `--selector`) also apply to other dbt tasks, such as `test` and `build`.
 
+
+<Tabs>
+<TabItem value="select" label="Examples for select flag">
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/syntax.md]:
```suggestion
<TabItem value="select" label="Examples of select flag">
```

File: website/docs/reference/node-selection/syntax.md [hunk 2]
```diff
@@ -52,21 +56,30 @@ The `--select` flag accepts one or more arguments. Each argument can be one of:
...
 dbt run --select "tag:nightly"           # run models with the "nightly" tag
 dbt run --select "path/to/models"        # run models contained in path/to/models
 dbt run --select "path/to/my_model.sql"  # run a specific model by its path
-  ```
+```
+
+</TabItem>
 
-dbt supports a shorthand language for defining subsets of nodes. This language uses the characters `+`, `@`, `*`, and `,`.
+<TabItem value="subset" label="Examples for subsets of nodes">
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/syntax.md]:
```suggestion
<TabItem value="subset" label="Examples of subsets of nodes">
```

File: website/docs/reference/node-selection/syntax.md [hunk 3]
```diff
@@ -52,21 +56,30 @@ The `--select` flag accepts one or more arguments. Each argument can be one of:
...
 
+dbt supports a shorthand language for defining subsets of nodes. This language uses the following characters:
 
-  ```bash
+- plus operator [(`+`)](/reference/node-selection/graph-operators#the-plus-operator)
+- at operator [(`@`)](/reference/node-selection/graph-operators#the-at-operator)
+- asterisk operator (`*`)
```
Comment (matthewshaver) [inline website/docs/reference/node-selection/syntax.md:76]: Do we have issues to create content for the * and , operators?

Comment (mirnawong1) [reply website/docs/reference/node-selection/syntax.md:76]: i dont think so, i was going to create an issue to add the asterisk and comma operator tomorrow, however if you want to tackle that - feel free!

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**